packages/apps/esm-help-menu-app/src/helpMenu/help-popup.component.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import styles from './help-popup.styles.scss';
+import GenericModal from './subHelpMenu/tutorial';
+import { useTranslation } from 'react-i18next';
+
+export default function HelpMenuPopup(props) {
+  const { t } = useTranslation();
+  const [tutorialsModalOpen, setTutorialsModalOpen] = useState(false);
+
+  const handleTutorialsModalOpen = () => {
+    setTutorialsModalOpen(true);
+  };
+
+  const handleTutorialsModalClose = () => {
+    setTutorialsModalOpen(false);
+  };
+
+  return (
+    <div className={styles.popup}>
+      <div className={styles.farRight}>
+        <GenericModal
+          isOpen={tutorialsModalOpen}
+          onClose={handleTutorialsModalClose}
+          heading="Tutorials"
+          label="Home"
+          content="Welcome to OpenMRS 3.0.1! Here are some tutorials to help you get started."
+        />
+        <button className={styles.helpButton} onClick={props.close}>
+          {t('releaseNotes', 'Release Notes')}
+        </button>
+        <button className={styles.helpButton} onClick={handleTutorialsModalOpen}>
+          {t('tutorials', 'Tutorials')}
+        </button>
+        <button className={styles.helpButton} onClick={props.close}>
+          {t('contactUs', 'Contact us')}
+        </button>
+        <button className={styles.helpButton} onClick={props.close}>
+          {t('docs', 'Docs')}
+        </button>
+        <button className={styles.helpButton} onClick={props.close}>
+          {t('feedbackForum', 'Feedback Forum')}
+        </button>
+        <button className={styles.helpButton} onClick={props.close}>
+          {t('version', 'OpenMRS 3.0.1')}
+        </button>
```

Comment:
@Vijaykv5 The extension slot should come here, right? Not the `help.component.tsx`
```suggestion
        <ExtensionSlot className={styles.helpMenuExtension} name="help-menu-slot"/>
```

Reply:
yeah that's right!
I thought the help menu **button** should also have to be an extension.

Reply:
@jayasanka-sack, just to clarify, we will be using extension slots in two scenarios, right?
1. To get the list of items in the help menu
2. To load the tutorials list

Reply:
By the way, if you do the suggested change now, we will have to create an extension for the `help-menu-slot` with `packages/apps/esm-help-menu-app/src/helpMenu/subHelpMenu/tutorial.tsx` component. Otherwise, there won’t be a way to open the tutorial modal.

Reply:
> just to clarify, we will be using extension slots in two scenarios, right?

@Piumal1999 yes